modules/rds-logs/variables.tf

```diff
@@ -133,6 +133,17 @@ variable "http_buffering_interval" {
   description = "Kinesis Firehose http buffer interval, in seconds."
 }
 
+variable "lambda_function_architecture" {
+  type        = string
+  default     = "arm64"
```

Comment:
If we make the default amd64 then this is probably a smoother upgrade path and could even be only a minor release of the module? 🤔

Reply:
I'm open to that! One of my main motivators to do arm was for the cost/performance benefits but tbh I don't know how significant those will be so if we think amd is better, I'll change it!

Reply:
I think they're great benefits, but, in the name of not surprising current consumers of the module I thought amd64 made sense as the default (as it's current behaviour)? Happy to push ARM64 here though ✨

Reply:
Agree, I think it's worth the tradeoff for the smoother upgrade path. I pushed the change!